usautobuild/api_caller.py

```diff
@@ -1,7 +1,8 @@
 import requests
 from logging import getLogger
 
-logger = getLogger("usautobuild")
+log = getLogger("usautobuild")
```

Comment:
What's the reason behind this semantic change?

Reply:
no strong reason for logger -> log. log is just a more commonly used name (not just in python). i can roll that change back if needed

Reply:
I don't have a strong inclination for either, so you can just leave it as it is now.